app/javascript/packages/session/requests.ts

```diff
@@ -83,20 +81,22 @@ function handleUnauthorizedStatusResponse(error: ResponseError) {
 /**
  * Request the current session status. Returns a promise resolving to the current session status.
  *
+ * @param sessionsUrl The URL for the session API
  * @return A promise resolving to the current session status
  */
-export const requestSessionStatus = (): Promise<SessionStatus> =>
-  request<SessionStatusResponse>(SESSIONS_URL)
+export const requestSessionStatus = (sessionsUrl: string): Promise<SessionStatus> =>
+  request<SessionStatusResponse>(sessionsUrl)
```

Comment:
we tend to prefer the capitalized acronyms in JS for variables (exceptions made for when reading the variables off of the `.dataset` directly)

```suggestion
export const requestSessionStatus = (sessionsURL: string): Promise<SessionStatus> =>
  request<SessionStatusResponse>(sessionsURL)
```

Reply:
Related docs: https://github.com/18F/identity-idp/blob/main/docs/frontend.md#naming-conventions

>All letters of an acronym should be capitalized, e.g. `const siteURL = 'https://example.com';`.